spring-web/src/main/java/org/springframework/http/server/reactive/WriteResultPublisher.java

```diff
@@ -117,7 +117,7 @@ public WriteResultSubscription(WriteResultPublisher publisher) {
 		}
 
 		@Override
-		public final void request(long n) {
+		public void request(long n) {
```

Comment:
Although `final` is technically unnecessary in the current arrangement, we prefer to make stylistic changes like this on our own when we deem it necessary.

So I'll revert those changes when merging.